js-miniapp-sample/src/pages/user-details.js

```diff
@@ -177,20 +179,22 @@ export const dataFetchReducer = (state: State, action: Action) => {
         ...state,
         isPointsLoading: true,
         isPointsError: false,
+        hasRequestedPermissions: false,
```

Comment:
It's also incorrect to set `hasRequestedPermissions` under `POINTS_FETCH_INIT`, `POINTS_FETCH_SUCCESS`, and `POINTS_FETCH_FAILURE` since these are the actions for points and not for user details.

Reply:
I have rechecked that we also need to set `hasRequestedPermissions: false` in here to fix the issue.

Reply:
Updated per suggestion.